src/sources/splunk_hec.rs

```diff
@@ -813,12 +811,8 @@ mod tests {
         rt: &mut Runtime,
     ) -> Vec<Event> {
         let n = messages.len();
-        assert!(
-            n <= CHANNEL_CAPACITY,
-            "To much messages for the sink channel"
-        );
         let pump = sink.send_all(stream::iter_ok(messages.into_iter().map(Into::into)));
-        let _ = rt.block_on(pump).unwrap();
+        rt.spawn(pump.map(|_| ()));
```

Comment:
`unwrap` not required in `spawn`?

Reply:
It would be nice. 
`.map_err(|()| panic!())` is not quite as effective, but it's closest that we can get.